### ISSUE TITLE
Hello,I would like to ask whether this method can achieve the results of the experiment in the paper？

### ISSUE THREAD

**Issue by Keeplingshi** [ThanhChinhBK/event_detector#1]
this paper 《Event Detection and Domain Adaptation with Convolutional Neural Network》

**Comment by ThanhChinhBK** [comment]
hi, Currently f1-score of this model is 0.65 with domain adaptation and unused name entity (NER), I will update model with name entity as soon as possible.

**Comment by Keeplingshi** [comment]
How much time does the model train?

**Comment by ThanhChinhBK** [comment]
about a day

**Comment by Keeplingshi** [comment]
Thank you very much

**Comment by duanshaoyang** [comment]
陈斌，你好！
       我想问一下你，他的代码你跑通了吗？我调适的时候遇到如下问题，想问你是如何解决的。望回复
InvalidArgumentError (see above for traceback): Nan in summary histogram for: b2_0/grad/hist
         [[Node: b2_0/grad/hist = HistogramSummary[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"](b2_0/grad/hist/tag, gradients/AddN)]]

**Comment by Keeplingshi** [comment]
跑通了，不过我没跑完。我只是想借用一下语料预处理的部分。
这个问题我不记得我有碰到过，如果有问题咱们私聊，我邮箱应该公开了。

**Comment by xiaoya6666** [comment]
您好，请问预处理部分的.rich_ere.xml文件指的是什么啊，我得到的ace数据里没有这部分的文件啊，是要经过处理得到的么？

**Comment by xiaoya6666** [comment]
您好，请问您的最后效果达到了论文水平么。现在召回率偏高，准确率很低，代码里的参数不是调整好的么？

**Comment by rainbow7xj** [comment]
@xiaoya6666 ,您好，请问您跑通这份代码了吗？

**Comment by xiaoya6666** [comment]
@rainbow7xj 通的。。。就是效果差大概20个点

**Comment by rainbow7xj** [comment]
@xiaoya6666 ,好的。感谢！
